xchange-core/src/test/java/org/knowm/xchange/dto/trade/StopOrderTest.java

```diff
@@ -85,6 +88,10 @@ public void testBuilderFrom() {
     final StopOrder copy = StopOrder.Builder.from(original).build();
 
     assertThat(copy).isEqualToComparingFieldByField(original);
+
+    StopOrder jsonCopy = ObjectMapperHelper.viaJSON(copy);
+    assertThat(jsonCopy).isEqualToIgnoringGivenFields(copy, "cumulativeAmount");
+    assertTrue(jsonCopy.getCumulativeAmount().compareTo(copy.getCumulativeAmount()) == 0);
```

Comment:
this mixes to units into one test

please separate and other tests like this

just build your own order and pass it viaJSON